src/main/com/yetanalytics/lrs/pedestal/routes.cljc

```diff
@@ -85,22 +85,38 @@
     :enter (fn health-fn [ctx]
              (assoc ctx :response {:status 200 :body ""}))}))
 
-(defn build [{:keys [lrs path-prefix]
-                     :or {path-prefix "/xapi"}}]
+(defn build
+  "Given a map with :lrs implementation, builds and returns xAPI routes
+  in pedestal table format.
+
+  Optional keys:
+    :path-prefix - defines the prefix from root for xAPI routes, default /xapi
+    :wrap-interceptors - a vector of interceptors to apply to every route"
+  [{:keys [lrs
+           path-prefix
+           wrap-interceptors]
+    :or {path-prefix "/xapi"
+         wrap-interceptors []}}]
```

Comment:
Now that I think about it, I think the `:wrap-interceptors` default should be `[error-interceptor]`, reason being that if someone totally forgets to set a error interceptor wrapper, they'll end up spewing the entire context map to log, which was the exact thing we were trying to avoid. In other words, better to provide some security than not.

Reply:
I can live with that

Reply:
https://github.com/yetanalytics/lrs/pull/71/commits/3365c2cab0cecf2fd62d2fed715afd5c8d7a1d03